src/le.py

```diff
@@ -1461,6 +1477,90 @@ def cancel(self):
         if self.timer:
             self.timer.cancel()
 
+class FollowMultiple(object):
+    """
+    The FollowMultiple is responsible for handling those logs that were set-up using the
+    '--multilog' option and that may have a wildcard in the pathname.
+    In which case multiple local (log) files will be followed, but with all the new events
+    from all the files forwarded to the same single log in the logentries infrastructure.
+    """
+
+    def __init__(self, name, entry_filter, entry_formatter, entry_identifier, transport, max_num_followers=MAX_FILES_FOLLOWED ):
+        """ Initializes the FollowMultiple. """
+        self.name = name
+        self.flush = True
+        self.entry_filter = entry_filter
+        self.entry_formatter = entry_formatter
+        self.entry_identifier = entry_identifier
+        self.transport = transport
+
+        self._shutdown = False
+        self._max_num_followers=max_num_followers
+        self._followers = []
+        self._worker = threading.Thread(
```

Comment:
I am not familiar with how the code works in general, however, do we need a thread here?